gutenberg.php

```diff
@@ -29,6 +29,13 @@ function the_gutenberg_project() {
 	global $post_type_object;
 	?>
 	<div class="block-editor gutenberg">
+		<noscript>
+			<div class="error" style='margin-top:32px'><p>
+			<?php
+			echo __( 'The block editor requires Javascript to be enabled.', 'gutenberg' );
```

Comment:
`echo __()` => `_e()`

Reply:
The `echo __()` function is used in numerous spots in gutenberg.php, I just mirrored it to stay consistent within the same file.

Reply:
Thumbs down is not really a productive response.
I've switched each of the occurrences to the shorter function.